src/tls/FFI.fst

```diff
@@ -108,3 +109,59 @@ let close_extraction_bug c : int =
   | writeClose                 -> 0
   | WriteError description txt -> errno description txt
   | _                          -> -1 
+
+  
+(* ************** Native FFI support  ************** *)
+
+let s2pv = function
+  | "1.2" -> TLS_1p2
+  | "1.3" -> TLS_1p3
+  | "1.1" -> TLS_1p1
+  | "1.0" -> TLS_1p0
+  | s -> failwith ("Invalid protocol version specified: "^s)
+
+let ffiConfig version host =
+  let v = s2pv version in 
+  {defaultConfig with
+    minVer = v;
+    maxVer = v;
+    check_peer_certificate = false;
+    cert_chain_file = "c:\\Repos\\mitls-fstar\\data\\test_chain.pem";
```

Comment:
figure these out dynamically? happy to expose more functions (e.g. getcwd()) from within F* if needed